pkg/e2e/verify/pr_check_queue.go

```diff
@@ -0,0 +1,72 @@
+package verify
```

Comment:
This should not be in `pkg/e2e` since it's not actually an e2e test.

This would be more appropriate in `pkg/common/`, maybe as part of the `cluster` package?

Reply:
Moved Over in the next commit.